cargo-afl/build.rs

```diff
@@ -94,6 +101,33 @@ fn build_afl_llvm_runtime(work_dir: &Path, base: Option<&Path>) {
     assert!(status.success());
 }
 
+fn build_afl_llvm_plugins(work_dir: &Path, base: Option<&Path>) {
```

Comment:
```suggestion
fn copy_afl_llvm_plugins(work_dir: &Path, base: Option<&Path>) {
```
Nit, since the pluigns were actually built in `build_afl`.

Reply:
Done in the latest commit